### DISCUSSION TITLE
Caricamento utenti (allievi, docenti, personale)

### DISCUSSION THREAD

**Discussion by giocampo** [iisgiua/giuaschool#529] [Richieste di aiuto] [answered]
Salve, l'istituto con cui collaboro ha più sedi dislocate sulla regione, e la sede principale raccoglie le informazioni di tutte per fini di rendicontazione.
E' possibile far fare l'inserimento di allievi, docenti e personale ata relativamente ad ogni sede o il tutto si può fare solo dalla sede centrale con l'account amministratore??

**Comment by trinko** [comment, marked as answer]
L'inserimento degli utenti è gestito solo dall'utente amministratore, non importa in che sede si trovi.
